### PR TITLE
test(pkg, mount): Add tests for getPartitionName in TestGetMountAttr

### PR DESCRIPTION
**Pull request by Z0Marlin**
Signed-off-by: Aditya Jain <aditya.jainadityajain.jain@gmail.com>

**Checklist:**
- [x] Fixes TODO in file  `mountutil_test.go`
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Has the change log section been updated? 
- [x] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

**Comment by Z0Marlin** [comment]
@akhilerm unit tests added as discussed in #590

File: pkg/mount/mountutil_test.go [hunk 1]
```diff
@@ -64,40 +69,87 @@ func TestGetMountAttr(t *testing.T) {
 
 	mountAttrTests := map[string]struct {
 		devPath           string
+		mountPoint        string
+		attrFunc          int
```
Comment (akhilerm) [inline pkg/mount/mountutil_test.go:73]: Why is this required? You can use the type [`getMountData`](https://github.com/openebs/node-disk-manager/blob/master/pkg/mount/mountutil.go#L44)

**Review by akhilerm — CHANGES_REQUESTED** [review]
@Z0Marlin Given some comments. Also, once this is merged, there will be changes in test cases in #590

File: pkg/mount/mountutil_test.go (hunk 1 above)
Comment (Z0Marlin) [reply pkg/mount/mountutil_test.go:73]: This is because the functions `getMountName` and `getPartitionName` are methods of `DiskMountUtil`. I can't reference them before creating the mountutil which is being created inside the test.

Comment (akhilerm) [reply pkg/mount/mountutil_test.go:73]: ok. got it.

**Review by akhilerm — APPROVED** [review]
LGTM

**Review by RealHarshThakur — APPROVED** [review]
